robot/power.py

```diff
@@ -43,6 +56,15 @@ def power_off(self):
         """
         self._send_and_receive({'power': False})
 
+    def set_output(self, output: PowerOutput, value: bool) -> None:
+        """
+        Turn off individual power output.
```

Comment:
What is our standard for docstrings? I see two different styles here

Reply:
Looks like we are inconsistent **everywhere**. This should probably be addressed in a separate PR

Reply:
Does the linter (`flake8-docstrings`) not tell you whether this is the expected format?

Reply:
Ah, it seems not; I'll add details on #98.